scalameta/contrib/src/main/scala/scala/meta/contrib/implicits/implicits.scala

```diff
@@ -0,0 +1,3 @@
+package scala.meta.contrib.implicits
+
+trait implicits extends Statements.ToStatementOps
```

Comment:
I like the idea of hiding implicits behind an optional `implicit` import. It seems to in convention with other libs such as cats/scalaz. If we're gonna go this route, we should move the Xtension methods in scala.meta.contrib package to the `implicits` too (I can do that in a separate PR)